package-lock.json

```diff
@@ -1,5340 +1,7 @@
 {
```

Comment:
I think that having a lockfile for both NPM and Yarn is dangerous. Conceivably, the two can get out of date, leading to the developer getting different package versions depending on whether they use NPM or Yarn.

Consider deleting one of the lockfiles and enforcing one of the two package managers in the readme.

Reply:
Yarn lives on